src/views/htmlcontent/tslint.json

```diff
@@ -60,7 +54,6 @@
     "no-string-literal": false,
     "no-switch-case-fall-through": false,
     "no-trailing-whitespace": true,
-    "no-unreachable": true,
```

Comment:
why do we want to remove this rule?

Reply:
This rule no longer exists in tslint - it's replaced by the compiler flag noImplicitReturns